### DISCUSSION TITLE
Discussions for Enterprise Cloud

### DISCUSSION THREAD

**Discussion by ap0phi5** [community/community#2932] [Enterprise] [answered]
Are we going to be able to use discussions on Enterprise Cloud anytime soon? Great that we can use it on public repos, but I would be keen to get it to help facilitate innersourcing in our org.

**Comment by becca** [comment, marked as answer]
Conversations for this can be found in https://github.com/github/feedback/discussions/2853 and https://github.com/github/feedback/discussions/40

**Comment by addityasingh** [comment]
@becca I believe the question is for Github Enterprise users. We use github enterprise at our organization and this feature is the exact solution we need right now to support internal teams with a community forum. Do you have a roadmap to enable this for enterprise users? 

Tagging @evi-liu based on the response in the [thread about Discussions on private repo](https://github.com/github/feedback/discussions/2853#discussioncomment-465695) 🙏
